### ISSUE TITLE
wording for the main README.md

### ISSUE THREAD

**Issue by fulverin** [01-edu/public#1640]
"

> Welcome to the Public Repository of the 01 Edu System
> 
> Our courses are meticulously studied in order to provide you with quality projects. Please take into account our approach before making Issues
> 
> Some feedback relates to the ambiguity of the subject-lack of precision, and this is voluntary on our part.
> 
> Do not hesitate to contribute by making a PR in the content-update branch.
> 
> Thank You 😄

"

https://github.com/01-edu/public

> Our courses are meticulously studied in order to provide you with quality projects. Please take into account our approach before making Issues

-> I do not really see how taking it into account should make a difference when posting any issue ...

> "Some feedback relates to the ambiguity of the subject-lack of precision, and this is voluntary on our part."

-> does it mean that some instructions are badly written but let's pretend it is part of the challenges to overcome and we will ignore any wording improvement request ?
If yes, then let's write it correctly, at least here, please.

Honestly I can understand we expect student to figure out things without too much specifications;
But we need to know if wording, or actual mathematical inaccuracies, have a chance to be changed.

**Comment by davhojt** [comment]
Hey @fulverin.

Thanks for your feedback.

Let's first say that the wording of that root readme is out-of-date, so it will be changed.

***

> I do not really see how taking it into account should make a difference when posting any issue

I agree with your point. As a matter of fact, I'd prefer to remove barriers to leaving feedback. I want to know more than just bad grammar. I want to know if you feel that some projects are not valuable to your learning journey, if the learning curve is too steep, if there are missed-opportunities to acquire deeper knowledge etc. Please do leave feedback. I mean that.

***

> does it mean that some instructions are badly written but let's pretend it is part of the challenges to overcome and we will ignore any wording improvement request ? If yes, then let's write it correctly, at least here, please.

We certainly do make improvements when we receive feedback. You will see that many of our closed issues have PRs attached. Most of the ones that don't have PRs attached, have no description of any problem.

Quite a lot of our content has been contributed by people who are not native English speakers. Occasionally, this means that something may not be as understandable as it was intended. Throughout Q1 of 2023, we will conduct a language review of all of our content - and implement processes to ensure that a language check is performed at the time of (or shortly after) any modifications to our content.

Occasionally, our subjects are intentionally vague. This is to encourage students to try to figure out the problem together, and to perform research. r/ProgrammerHumour is full of examples of vague product requirements 😅. Trust me, it is a good skill to be able to operate with little information.

I think the problem is more like you said: that it is difficult to tell if something is intentionally vague, or if something is unintentionally poorly written. As I said before, we will review all of the content in Q1 of 2023. It's very easy for us to make changes, and we do read all feedback from students. So in any case that we don't make a change in response to the feedback, it is likely because we feel that the piece of content in question is written as it was intended.

Finally, of course individuals have different points of view. So you are very welcome to tag me if you concretely disagree with a decision. That of course isn't a guarantee that the content will be changed, but at least you'll have some piece of mind that more consideration has been given.

Which school are you at by the way?

**Comment by fulverin** [comment]
Hi,
sorry for not replying, I posted this while logged with wrong account, so I did not catch the notifications.
I am at kood/Johvi (estonia) 
By the time I realized the PR I posted have actually been taken into account, thanks for that.

By the way may be I wasn't curious/listening  but it is not that straightforward for me how things are subdivided:
is staff from my school involved into 01edu platform maintenance ? or is it purely maintained by the 01edu team ?
May be it would worth mentioning something about it so we have an idea of who we are talking with.

Your reply is exactly what I would expect to find in the first README.

Thanks you for clarification.
